### PR TITLE
Design

### PR DESCRIPTION
**Pull request by lindaicing**
Combining Design (styling of css) and removed unnecessary files (such as an unupdated manifest.json and a placeholder icon)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+Extension/.DS_Store
```
Comment (CubeOfCheese) [inline .gitignore:2]: .gitignore should not be pushed to github. Add .gitignore to the .gitignore file

Comment (lindaicing) [reply .gitignore:2]: How do I find the .gitignore?

File: Extension/popup.html [hunk 2]
```diff
@@ -30,7 +24,7 @@ <h2 id="overallScore">8.5</h2>
                 <div>
                     <span>
                         <span class="scorebar"></span>
-                        <span class="scorebarFill"></span>
+                    <span class="scorebarFill"></span>
```
Comment (CubeOfCheese) [inline Extension/popup.html:27]: Why was indent removed?

Comment (lindaicing) [reply Extension/popup.html:27]: Oh hm I'm not sure! It shouldn't be though. Will fix!

**Comment by lindaicing** [comment]
> Looks good. In the future, please include a screenshot with your pull request to help visualize the changes

Will do! Here's what it looks like now.
Screen Shot 2020-05-27 at 2 00 03 PM

**Pull request merged**